README.md

```diff
@@ -1 +1,31 @@
-# E-commerce-app-team1
\ No newline at end of file
+# E-commerce-app-team1
+
+### Why we did this? 
+- This app was designed and implemented to make people's lifes easier selling and buying things online. 
+- This is the final project from the Fundamentals program provided by Gaza Sky Geeks (GSG). 
+- It aims at developing our web development skills and gaining new skills other than those acquired during the program. 
+---
+
+### What is this app? 
```

Comment:
you can say that is User Journey

Reply:
You mean replace (what is this app?) with (User Journey)? If not, please elaborate

Reply:
yes that what I mean

Reply:
Alright, I'll do it. But what to write in the (what) section?

Reply:
you can say its platform to sell or buy  products

Reply:
Done.